vars/common-vars.yml

```diff
@@ -32,6 +32,20 @@ socok8s_repos_to_configure:
   openSUSE-Leap-15.0-Cloud: "{{ obs_mirror }}/repositories/Cloud:/Tools/openSUSE_Leap_15.0/"
   Leap15develtools: "{{ obs_mirror }}/repositories/devel:/tools/openSUSE_Leap_15.0/"
   socok8s: "{{ obs_mirror }}/repositories/Cloud:/socok8s:/master/openSUSE_Leap_15.0/"
+  SLES15-JeOS-OpenStack-Cloud-GM-Pool: http://download.suse.de/ibs/SUSE:/SLE-15:/GA/standard/
```

Comment:
Should we start to split this? Eg. introduce `socok8s_deployer_repos` or something like this?

Reply:
it may make sense if we want to support multiple OS for the deployer

Reply:
I was more thinking that the variable `socok8s_repos_to_configure` might be used in multiple places (on different hosts). But if we anyway use it only on the deployer, than fine for me to just add more repos..